File: src/main/java/com/bajdcc/LALR1/interpret/os/ui/UIClock.java
package com.bajdcc.LALR1.interpret.os.ui;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【界面】时钟
 *
 * @author bajdcc
 */
public class UIClock implements IOSCodePage {
	@Override
	public String getName() {
		return "/ui/clock";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/java/com/bajdcc/LALR1/interpret/os/ui/UIClock.kt
package com.bajdcc.LALR1.interpret.os.ui

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【界面】时钟
 *
 * @author bajdcc
 */
class UIClock : IOSCodePage {
    override val name: String
        get() = "/ui/clock"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
